app/views/layouts/application.html.erb

```diff
@@ -2,13 +2,16 @@
 <html>
 <head>
   <title>OpportunityAtWork</title>
+  <!-- if the scripts are in the head, browser will load scripts first before the body.
+  If the scripts are last, then it displays the page first, then load the javascript. -->
```

Comment:
I think these comments were left in by mistake. Seems like we should be able to remove them but wanted to make sure before I did!